config/datatypes_conf.xml.sample

```diff
@@ -736,6 +736,7 @@
     <datatype extension="dada2_mergepairs" type="galaxy.datatypes.binary:RData" subclass="true" display_in_upload="true" />
     <datatype extension="dada2_sequencetable" type="galaxy.datatypes.tabular:Tabular" mimetype="application/text" subclass="true" display_in_upload="true" />
     <datatype extension="dada2_uniques" type="galaxy.datatypes.tabular:Tabular" mimetype="application/text" subclass="true" display_in_upload="true" />
+    <datatype extension="ckpt" type="galaxy.datatypes.binary:Tensorflow" display_in_upload="true" />
```

Comment:
Since the class in binary.py it's basically empty, you can remove it and use `subclass="true"` like the `cxb` datatype.

Reply:
Awesome, thanks!